pkg/krmtotf/tftokrm.go

```diff
@@ -233,7 +234,16 @@ func ResolveSpecAndStatusWithResourceID(resource *Resource, state *terraform.Ins
 func resolveDesiredStateInSpecAndObservedStateInStatus(resource *Resource, state *terraform.InstanceState) (
 	spec map[string]interface{}, status map[string]interface{}) {
 	spec = deepcopy.MapStringInterface(resource.Spec)
-	_, status = GetSpecAndStatusFromState(resource, state)
+	returnedSpec, status := GetSpecAndStatusFromState(resource, state)
+	if resource.Kind == "DataflowJob" {
```

Comment:
nit: we can add a comment of this special handling in the code for future reference.

Reply:
Done.